src/components/VLanguage/VLanguage.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <VSelectField
+    v-bind="$attrs"
+    v-model="locale"
+    field-id="language"
+    :choices="choices"
+    :blank-text="$t('language.language')"
+    :label-text="$t('language.language')"
+  >
+    <template #start>
+      <VIcon :icon-path="globeIcon" />
+    </template>
+  </VSelectField>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent } from '@nuxtjs/composition-api'
+
+import { useI18n } from '~/composables/use-i18n'
+
+import VIcon from '~/components/VIcon/VIcon.vue'
+import VSelectField, {
+  type Choice,
+} from '~/components/VSelectField/VSelectField.vue'
+
+import type { LocaleObject } from '@nuxtjs/i18n'
+
+import globeIcon from '~/assets/icons/globe.svg'
+
+/**
+ * We put a little extra information in the Vue-i18n `locales` field such as the
+ * locale's name and native name, which comes in use here.
+ */
+interface OpenverseLocale extends LocaleObject {
+  name: string
+  nativeName?: string
+}
+
+/**
+ * Presents a way for the users to change the app locale and use a translated
+ * version of the app.
+ */
+export default defineComponent({
+  name: 'VLanguage',
```

Comment:
Can we add `Select` to the end of this name to clarify the purpose of the component?

Reply:
I used the exact name that is being used to refer to this component in Figma. The generic version of the component is named `VSelectField` to signify it's purpose.

Reply:
I prefer the explicitness of `VLanguageSelect` too; if we want parity with figma it might make sense to rename in Figma _or_ to link to the [Figma component definition](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=2960%3A8532) in the comments for this component.

Reply:
Renamed in 9a992f4c.